toxav/video.c

```diff
@@ -34,6 +34,8 @@
 #include "../toxcore/network.h"
 
 #define MAX_DECODE_TIME_US 0 /* Good quality encode. */
+#define MAX_ENCODE_TIME_US ((1000 / 24) * 1000)
```

Comment:
1. It moved from toxav.c as is
2. As far as I understand it is a hint for encoder to try maintain 24 fps performance
3. Not sure if it is good idea to have this hardcoded but it out of this PR scope